packages/wonder-blocks-birthday-picker/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "@khanacademy/wonder-blocks-birthday-picker",
+    "version": "0.0.1",
```

Comment:
Since it'll get bumped on release, should we start new packages out at `0.0.0` instead?